lib/peep.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+
+class Peep
+
+  def self.add(name:, handle:, post:)
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
Did you think about creating a DatabaseConnection class? This would help tidy up the code here to align more with SRP.